help/sources/tutorials/ui/create/data-partners/acxiom-prospecting-data-import.md

```diff
@@ -0,0 +1,154 @@
+>
+>* To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+>This tutorial requires a working understanding of the following components of Experience Platform:
+
+* [[!DNL Experience Data Model (XDM)] System](../../../../../xdm/home.md): The standardized framework by which Experience Platform organizes customer experience data.
+  * [Basics of schema composition](../../../../../xdm/schema/composition.md): Learn about the basic building blocks of XDM schemas, including key principles and best practices in schema composition.
+  * [Schema Editor tutorial](../../../../../xdm/tutorials/create-schema-ui.md): Learn how to create custom schemas using the Schema Editor UI.
+* [[!DNL Real-Time Customer Profile]](../../../../../profile/home.md): Provides a unified, real-time consumer profile based on aggregated data from multiple sources.
+
+>[!IMPORTANT]
+>
+>To connect to the source, you need the **[!UICONTROL View Sources]** and **[!UICONTROL Manage Sources]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+
+# Create a [!DNL Acxiom Prospecting Data Import] source connection and dataflow in the UI
+
+>[!NOTE]
+>
+>The [!DNL Acxiom Prospecting Data Import] source is in beta. Please read the [sources overview](../../../../home.md#terms-and-conditions) for more information on using beta-labeled sources.
+
+### Gather required credentials
+
+In order to access your bucket on Experience Platform, you need to provide valid values for the following credentials:
+
+| Credential | Description |
+| --- | --- |
+| Acxiom authentication key | The authentication key. You can retrieve this value from the [!DNL Acxiom] team. |
+| S3 Access key | The access key ID for your bucket. You can retrieve this value from the [!DNL Acxiom] team. |
+| S3 Secret key | The secret key ID for your bucket. You can retrieve this value from the [!DNL Acxiom] team. |
+| Bucket name | This is your bucket where files will be shared. You can retrieve this value from the [!DNL Acxiom] team. |
```

Comment:
Let's move this section to the Prerequisite section. Right after the list of understanding other Platform concepts

Reply:
I think I have this moved to the correct location now.  Please let me know if I misinterpreted the request.  Thanks